mms/model_loader.py

```diff
@@ -137,15 +137,23 @@ def _extract_model(service_name, path):
 
     assert len(glob.glob(os.path.join(model_dir, manifest['Model']['Signature']))) == 1, \
         'Signature file in model archive is inconsistent with manifest.'
+    print("INFO: Signature file configured")
 
-    assert len(glob.glob(os.path.join(model_dir, manifest['Model']['Symbol']))) == 1, \
-        'Symbol file in model archive is inconsistent with manifest.'
+    try:
+        if len(glob.glob(os.path.join(model_dir, manifest['Model']['Symbol']))):
+            print("INFO: Symbol file configured")
+    except Exception:  # pylint: disable=broad-except
+        print("WARNING: Symbols file not given.")
```

Comment:
Why always show a warning? if the model archive is imperative, we do not expect symbol and it is fine. Can we change this to warn only if it actually needs the symbol?

Reply:
True, we can read the model-format and decide if this should be an error or a info.